scripts/provisioning/s3_haproxy_config.py

```diff
@@ -358,11 +368,12 @@ def configure_haproxy_k8(self):
     pem_dir = os.path.dirname(sslcertpath)
     if not os.path.exists(pem_dir):
         os.makedirs(pem_dir)
-    if not os.path.exists('/etc/cortx/haproxy/errors/'):
-        os.makedirs('/etc/cortx/haproxy/errors/')
+    errors_path = os.path.join(baseconfig_path, 'haproxy/errors/')
+    if not os.path.exists(errors_path):
+        os.makedirs(errors_path)
 
     #Run config commands
-    os.system("cp /opt/seagate/cortx/s3/install/haproxy/503.http /etc/cortx/haproxy/errors/")
+    shutil.copy('/opt/seagate/cortx/s3/install/haproxy/503.http', errors_path)
```

Comment:
use this correct path in the config itself, line 271

```
errorfile               503 /etc/haproxy/errors/503.http
```

Reply:
Done.